src/library-authoring/list-libraries/index.scss

```diff
@@ -1,134 +1,155 @@
+    margin-bottom: 10px;
   }
 }
+
+// /* Adapted from:
+//  * edx-platform/cms/static/sass/views/_dashboard.scss
+//  */
+// .library-list-wrapper {
+//   margin: 0;
+//   // font-size: 1.4rem;
+
+//   .library-list-pagination {
+//     display: block;
+//     margin-top: 1.5em;
+
+//     * {
+//       font-size: 1.4rem;
+//     }
+
+//     .pagination {
+//       justify-content: center;
+//       .page-item {
+//         .btn {
+//           min-width: 2.5em;
+//           min-height: 2.5em;
+//         }
+//       }
+//     }
+//   }
+
+//   .library-list {
+//     border-radius: ($baseline*0.3);
+//     background: $white;
+//     padding: 0;
+//     margin: 0;
+
+//     .library-item.no-hover-bg {
+//       &:hover {
+//         background: none;
+//       }
+//     }
+
+//     .library-item {
+//       @extend %ui-depth2;
+//       @include margin-right(2%);
+
+//       box-sizing: border-box;
+//       width: 100%;
+//       position: relative;
+//       border: 1px solid $light-700;
+//       border-radius: ($baseline*0.3);
+//       box-shadow: none;
+//       outline: none;
+//       padding: ($baseline*1.20);
+//       display: inline-block;
+//       vertical-align: middle;
+      
+//       .pgn__card-header {
+//         padding: 0;
+
+//         &-content {
+//           margin-top: 0;
+//         }
+//         &-title-md {
+//           @extend %t-title4;
+//           @extend %t-bold;
+//           @include margin(0, ($baseline*2), ($baseline/1.25), 0);
+          
+//           color: $primary-500;
+//         }
+//       }
+
+//       .library-title {
+//         @extend %t-title4;
+//         @extend %t-bold;
+
+//         @include margin(0, ($baseline*2), ($baseline/4), 0);
+//         @include font-size(18);
+//         @include line-height(24);
+//       }
+
+//       .library-metadata {
+//         @extend %t-copy-sub1;
+
+//         @include transition(opacity $tmg-f1 ease-in-out 0);
+
+//         color: $gray-d2;
+
+//         .metadata-item {
+//           display: inline-block;
+
+//           & + .metadata-item::before {
+//             content: "•";
+//             margin-left: ($baseline/4);
+//             margin-right: ($baseline/4);
+//             color: $gray-500;
+//           }
+
+//           .label {
+//             @extend %cont-text-sr;
+//           }
+//         }
+//       }
+
+//       &.is-processing {
+//         .course-status .value {
+//           color: $gray-l2;
+//         }
+//       }
+
+//       &.has-error {
+//         .course-status {
+//           color: $red;
+//         }
+
+//         ~ .status-message {
+//           background: $red-l1;
+//           color: $white;
+//         }
+//       }
+
+//       &:not(:last-child) {
+//         margin-bottom: ($baseline*0.4);
+//       }
+//     }
+//   }
+
+//   .filter-form {
+//     .form-label {
+//       font-weight: 600;
+//     }
+
+//     button,
+//     .form-control {
+//       font-size: 1.4rem;
+//     }
+//   }
+// }
```

Comment:
Reminder to get rid of the saved reference.

Reply:
addressed in https://github.com/openedx/frontend-app-library-authoring/commit/42c82d41cd835ecf990db9a694c02cf19983f998